arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1965,5 +1965,11 @@
 			</dict>
 		</array>
 	</dict>
+	<dict>
+		<key>displayName</key>
+		<string>Favorites</string>
+		<key>children</key>
+		<array/>
+	</dict>
```

Comment:
Putting the category first is a great idea! Although, I disagree with dynamically including the favorites category. I think it could be confusing for the user. For example, if a user forgets that they have no favorited samples remaining or if they unknowingly unfavorited the sample on accident. They may wonder why they can't access their favorites anymore. Instead, we can put placeholder text when the view controller is empty.

Reply:
I like the idea of adding the favorites at the top of the list since a benefit of having a favorites category is having quick access to specific samples. But I will add text when there are no favorited samples.